src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/UpdateInstanceStatusV6.sql

```diff
@@ -34,7 +36,8 @@ CREATE OR ALTER PROCEDURE dbo.UpdateInstanceStatusV6
     @sopInstanceUid     VARCHAR(64),
     @watermark          BIGINT,
     @status             TINYINT,
-    @maxTagKey          INT = NULL
+    @maxTagKey          INT = NULL,
```

Comment:
Should have a new version of this sproc since we are updating the parameters.

Reply:
Compat is maintained without sproc version, because it has the default.